stylesheets/_component.forms.scss

```diff
@@ -36,23 +35,25 @@ fieldset {
         }
     }
 
+    &[type="password"],
+    &[type="text"] {
```

Comment:
Could we have classes for these instead, as we've got them for textareas, selects, checkboxes and radios already? Ideally those we already have would become modifiers of form__control instead of their own class, maybe something to iterate in?